sru-templates/manual/azure-sru

```diff
@@ -123,220 +109,219 @@ az network nsg rule create \
     --destination-port-ranges "*"
 
 az network vnet create \
-    --resource-group $RESOURCE_GROUP \
-    --location $REGION \
-    --name $VNET_NAME \
+    --resource-group $AZURE_RESOURCE_GROUP \
+    --location $AZURE_REGION \
+    --name $AZURE_VNET_NAME \
     --address-prefixes "10.0.0.0/16" "ace:cab:deca::/48"
 
 az network vnet subnet create \
     --name sruSubnet \
-    --resource-group $RESOURCE_GROUP \
-    --vnet-name $VNET_NAME \
+    --resource-group $AZURE_RESOURCE_GROUP \
+    --vnet-name $AZURE_VNET_NAME \
     --address-prefixes "10.0.0.0/24" "ace:cab:deca:deed::/64" \
-    --network-security-group $NETWORK_SECURITY_GROUP
+    --network-security-group $AZURE_NETWORK_SECURITY_GROUP
 
 
 sshopts="-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR"
-for series in $SRU_SERIES; do
```

Comment:
I am probably missing something here, but why are we dropping this for loop ? For what I understood by the new code, we now need to manually change the `UBUNTU_RELEASE` parameter to test releases. But wouldn't this cause the process to be slower ?

Reply:
@lucasmoura the for loop should be dropped because we expect to run these tests in parallel in separate terminals instead of serially in a single for loop. In a followup PR I'm going to adjust azure-sru specifically to better allow for idempotent creation of shared resource groups etc so that the different scripts will not collide during creation of shared resources.

Reply:
That makes sense, thank @blackboxsw